nemo/collections/nlp/models/token_classification/punctuation_capitalization_model.py

```diff
@@ -98,14 +106,18 @@ def __init__(self, cfg: DictConfig, trainer: Trainer = None):
         self.setup_optimization(cfg.optim)
 
     @typecheck()
-    def forward(self, input_ids, token_type_ids, attention_mask):
+    def forward(self, input_ids, attention_mask, token_type_ids=None):
         """
         No special modification required for Lightning, define it as you normally would
         in the `nn.Module` in vanilla PyTorch.
+        token_type_ids are not used for DistilBert models
         """
-        hidden_states = self.bert_model(
-            input_ids=input_ids, token_type_ids=token_type_ids, attention_mask=attention_mask
-        )
+        if 'distilbert' in self.model_cfg.language_model.pretrained_model_name:
```

Comment:
Do we need this? Passing None for token_type_ids isn't OK?

Reply:
changed the distilbert module